### DISCUSSION TITLE
Use GitHub Copilot with GitHub Enterprise

### DISCUSSION THREAD

**Discussion by Alex2497** [community/community#52562] [Enterprise]
## I want to use **GitHub Copilot** with my account **GitHub Enterprise**.

I installed **GitHub Copilot**:

![image](https://user-images.githubusercontent.com/36898509/231291855-e9e62102-f600-4ed0-a8f4-27173b6776e0.png)

I received a message to login:

![image](https://user-images.githubusercontent.com/36898509/231292018-ed11006f-50b1-42ee-82a5-fe4992f745c1.png)

The issue is that I want to login with my GitHub Enterprise account but this link are sending me to the normal login:

![image](https://user-images.githubusercontent.com/36898509/231292636-4bb6114b-ec0e-4666-a8fe-2b35c38ad9e7.png)

Any idea how I can access with my GitHub Enterprise Account?

**Comment by ArivuLabs** [comment]
The same issue with me as well; I wanted to login into the enterprise link instead the github.com.

Does anyone have any idea?
